include/xtensor/xcsv.hpp

```diff
@@ -29,7 +29,7 @@ namespace xt
     using xcsv_tensor = xtensor_container<std::vector<T, A>, 2, layout_type::row_major>;
 
     template <class T, class A = std::allocator<T>>
-    xcsv_tensor<T, A> load_csv(std::istream& stream);
+    xcsv_tensor<T, A> load_csv(std::istream& stream, const std::string comments = "#", const char delimiter = ',', const unsigned long skip_rows = 0, const long long max_rows = -1);
```

Comment:
maybe we can use std::size_t and std::ptrdiff_t for the arguments like elsewhere.